src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public void setOperation(Operation operation) {
+        this.operation = operation;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
```

Comment:
Not sure you need setters here. Have you used them?

Reply:
Yes)